src/test/ui/borrowck/borrowck-closures-slice-patterns.stderr

```diff
@@ -75,7 +75,7 @@ LL | fn arr_box_by_move(x: Box<[String; 3]>) {
 LL |     let f = || {
    |             -- value moved into closure here
 LL |         let [y, z @ ..] = *x;
-   |                            - variable moved due to use in closure
+   |                           -- variable moved due to use in closure
```

Comment:
Is this the only test case impacted? From my understanding of the changes, it seems like at least new test cases should be introduced to confirm the behavior is now different (if none of the other one are currently impacted).

Reply:
this change goes in before the new behavior

Reply:
Also the new behavior doesn't affect things yet